### PR TITLE
installation: Add guide for using s390x developer builds

### PR DESCRIPTION
**Pull request by jschintag**
**What this PR does / why we need it**:

Add instructions on how to use the nightly builds for s390x architecture.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
I have considered making the arm64 section more generic instead, but ultimately decided it would make the instructions needlessly compilecated.

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [x] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [x] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [x] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [x] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
Add instructions for using kubevirt s390x developer builds
```

### CODE REVIEW

File: docs/cluster_admin/installation.md [hunk 1]
```diff
@@ -225,6 +225,14 @@ ARM64 developer builds can be installed like this:
     $ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-operator-arm64.yaml
     $ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-cr-arm64.yaml
 
+### s390x developer builds
+
+s390x developer builds can be installed like this:
```
Comment (alicefr) [inline docs/cluster_admin/installation.md:230]: nit: `installed like this` -> `in this way`

**Comment by alicefr** [comment]
One small nit, otherwise looks good

File: docs/cluster_admin/installation.md (hunk 1 above)
Comment (jschintag) [reply docs/cluster_admin/installation.md:230]: I copied this directly from arm64, should i change it there as well?

**Pull request merged**